main.go

```diff
@@ -123,17 +142,131 @@ func NewExporter(server string, timeout time.Duration) *Exporter {
 			nil,
 		),
 		evictions: prometheus.NewDesc(
-			prometheus.BuildFQName(namespace, "", "items_evicted_total"),
+			prometheus.BuildFQName(namespace, "", "evicted_total"),
 			"Number of valid items removed from cache to free memory for new items.",
 			nil,
 			nil,
 		),
 		reclaimed: prometheus.NewDesc(
-			prometheus.BuildFQName(namespace, "", "items_reclaimed_total"),
+			prometheus.BuildFQName(namespace, "", "reclaimed_total"),
 			"Number of times an entry was stored using memory from an expired entry.",
 			nil,
 			nil,
 		),
+		active_slabs: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "", "active_slabs_total"),
+			"Total number of slab classes allocated.",
+			nil,
+			nil,
+		),
+		malloced: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "", "malloced_bytes"),
+			"Number of bytes of memory allocated to slab pages.",
+			nil,
+			nil,
+		),
+		itemsNumber: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "", "items_number_total"),
+			"Total number of items currently stored in this slab class.",
+			[]string{"slab"},
+			nil,
+		),
+		itemsAge: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "", "items_age_seconds"),
+			"Number of seconds the oldest item has been in the slab class.",
+			[]string{"slab"},
+			nil,
+		),
+		itemsEvicted: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "", "items_evicted_total"),
```

Comment:
With my suggestion above, this would become `memcached_slab_items_evicted_total{number="x"}` and not longer conflict with the overall `memcached_items_evicted_total`.